src/lib/intent/tag-create.ts

```diff
@@ -0,0 +1,44 @@
+import Intent from './base-intent'
+import { TagCreateAction } from '../action/tag-create'
+import chalk from 'chalk'
+import { PlanMessage } from '../interfaces/plan-message'
+
+export default class TagCreateIntent extends Intent {
+  isTagIntent () {
+    return true
+  }
+
+  isTagCreate () {
+    return true
+  }
+
+  getTagId () {
+    return this.payload.tagId
+  }
+
+  groupsWith (): boolean {
+    return false
```

Comment:
I guess from seeing this `groupsWith` actually only says whether the current one should be grouped with the previous one. If that's the case, maybe we should rename it to `groupsWithPrevious` or something like that.  

Just an idea, maybe @zcei has some opinions on this and remembers our design choices? 😄

Reply:
This makes sense in my opinion, I am just a bit reluctant to make it part of this pr as it will modify another 20 files. I would suggest that I do this renaming afterwards in a separate pr?

Reply:
But if you disagree, I am also happy to change it and bundle everything together!

Reply:
Nope works for me.